lib/tasks/sequel_management.rake

```diff
@@ -30,7 +30,10 @@ class SequelManagement
 
         logger.info "Begin rolling back new migrations"
 
-        original_migrations = `#{git_command}`.split.map { |path| File.basename(path) }
+        migration_files = `#{git_command}`
+        abort "Can't get list of migration files" unless $?&.success?
```

Comment:
Let's use `$CHILD_STATUS` to make it a bit more clear. Also why do we need safe navigation here?

Reply:
How about use `Open3.capture3` to simplify this condition? This solutions should look better than `$?` and `$CHILD_STATUS` (see last commit).